x-pack/plugins/security_solution/server/usage/queries/utils/get_event_log_agg_by_rule_type_metrics.ts

```diff
@@ -69,6 +69,16 @@ export const getEventLogAggByRuleTypeMetrics = (
           field: 'kibana.alert.rule.execution.metrics.total_search_duration_ms',
         },
       },
+      maxTotalEnrichmentDuration: {
+        max: {
+          field: 'kibana.alert.rule.execution.metrics.total_enrichment_duration_ms',
+        },
+      },
+      minTotalEnrichmentDuration: {
+        min: {
+          field: 'kibana.alert.rule.execution.metrics.total_enrichment_duration_ms',
```

Comment:
Hey @nkhristinin, is there a reason why we are not calculating `average` for this metric like we do for the others?

Reply:
Thanks for catching it, I added it